packages/x-components/tests/support/index.ts

```diff
@@ -14,8 +12,9 @@ import Loggable = Cypress.Loggable;
 import Shadow = Cypress.Shadow;
 import Timeoutable = Cypress.Timeoutable;
 import Withinable = Cypress.Withinable;
+import CommandFns = Cypress.CommandFns;
 
-interface CustomCommands {
+interface CustomCommands extends CommandFns {
```

Comment:
Extending the CommandFns interface so we ensure all functions are compatible commands.